backend/satellite_tools/reposync.py

```diff
@@ -35,6 +35,7 @@
 import multiprocessing
 
 from rhn.connections import idn_puny_to_unicode
+from rhn.i18n import ustr
```

Comment:
```suggestion
```
`ustr()` just calls `str()` in Python 3, we can use it directly in Python3-only code

Reply:
but `str()` does not work

Reply:
Oh, that's probably because in `ustr` we ignore decoding errors.
```python
str(obj, 'utf8', errors='ignore')
```
That might a bad thing because ignoring means skipping "characters". On the other hand, if the `str` is only used to verify dates, it shouldn't make a difference if we ignore errors or handle them e.g. via `errors='replace'`.

Reply:
No, it had something to do with "newline" (\n).
When using str() I was not able to call ".split()" on the result. 
the \n was escaped and it still result in 1 line.

Doing nothing let ".split()" fail.

Only with ustr() I got the result I wanted.

Reply:
How can I debug this issue?

It's very surprising that you get this behavior, calling `str()` directly or through `ustr()` should not make a difference in that regard.